python/cugraph/bsp/traversal/bfs_bsp.py

```diff
@@ -127,7 +127,8 @@ def bfs_df_pregel(_df, start, src_col='src', dst_col='dst', copy_data=True):
         aggsOut = OrderedDict()
         aggsOut['predecessor'] = 'min'
         aggsOut['distance'] = 'min'
-        _a = one_hop.groupby(['vertex'], as_index=False).agg(aggsOut)
+        _a = one_hop.groupby(
```

Comment:
this doesn't really need to be sorted - but it is okay since we should consider dropping the BSP folder in 0.19